From c431fe992685367f7ae8c6ea2022936612f5faae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Roland=20P=C3=A1kai?= <roland.pakai@liferay.com>
Date: Tue, 21 May 2019 09:38:15 +0200
Subject: [PATCH] LPS-95472 Tabs in popups not appears correctly in maximized
 CKEditor

Use `width: inherit` instead of `width: 100%` to work with the maximize
plugin.
---
 plugins/dialog/plugin.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/dialog/plugin.js b/plugins/dialog/plugin.js
index 9a8d6fb20..8a94776f5 100644
--- a/plugins/dialog/plugin.js
+++ b/plugins/dialog/plugin.js
@@ -1184,7 +1184,7 @@ CKEDITOR.DIALOG_STATE_BUSY = 2;
 					children: contents.elements,
 					expand: !!contents.expand,
 					padding: contents.padding,
-					style: contents.style || 'width: 100%;'
+					style: contents.style || 'width: inherit;'
 				}, pageHtml ),
 				contentMap = this._.contents[ contents.id ] = {},
 				cursor,
